ppb/engine.py

```diff
@@ -87,5 +111,8 @@ def activate(self, next_scene: dict):
         self.scenes.append(scene(self, *args, **kwargs))
 
     def update_input(self):
+        """
+        Updates mouse state
```

Comment:
Phrase as a command, also, upate will also update the keyboard object in the future, so "update input devices" is a more generic way to phrase this.